node/tessel-export.js

```diff
@@ -578,7 +585,8 @@ Tessel.Pin.prototype.addListener = function(mode, callback) {
 Tessel.Pin.prototype.on = Tessel.Pin.prototype.addListener;
 
 Tessel.Pin.prototype._setInterruptMode = function(mode) {
-  this.interruptMode = mode;
+  // rise and fall events will be emitted by change event
+  (mode === 'rise' || mode === 'fall') ? this.interruptMode = 'change' : this.interruptMode = mode;
```

Comment:
>Although, isn't it more nice to throw an error if mode is anything but undefined, 'rise', 'fall', 'change'?

Based on the question from @LinusU about throwing an error if the `mode` is invalid, should the `.addListener()` function be structured like this?

```javascript
Tessel.Pin.prototype.addListener = function(mode, callback) {
  if (typeof Tessel.Pin.interruptModes[mode] !== 'undefined') {
    if (!this.interruptSupported) {
      throw new Error(`Interrupts are not supported on pin ${this.pin}. Pins 2, 5, 6, and 7 on either port support interrupts.`);
    }

    // For one-time 'low' or 'high' event
    if ((mode === 'low' || mode === 'high') && !callback.listener) {
      throw new Error('Cannot use "on" with level interrupts. You can only use "once".');
    }

    // Can't set multiple listeners when using 'low' or 'high'
    if (this.interruptMode) {
      var singleEventModes = ['low', 'high'];
      if (singleEventModes.some(value => mode === value || this.interruptMode === value)) {
        throw new Error(`Cannot set pin interrupt mode to ${mode}; already listening for ${this.interruptMode}. Can only set multiple listeners with change, rise & fall`);
      }
    }

    // Set the socket reference so the script doesn't exit
    this._port.ref();
    this._setInterruptMode(mode);

    // Add the event listener
    Tessel.Pin.super_.prototype.on.call(this, mode, callback);
  } else {
    throw new Error(`Invalid pin event mode '${mode}'. Valid modes are 'change', 'rise', 'fall', 'high' and 'low'.`);
  }
};
```